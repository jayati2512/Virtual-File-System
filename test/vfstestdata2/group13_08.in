createvfs database 1024
mountvfs database
mountvfs database
makedir /westernclothes
makedir /westernclothes jeans
makedir /westernclothes shirts
makedir /westernclothes t-shirt
makedir /indianclothes
makedir /indianclothes saree
makedir /indianclothes suits
makedir /indianclothes lehngas
movedir /indianclothes/lehngas /westernclothes
deletedir /westernclothes
addfile /indianclothes/saree red.txt /user/include
addfile /indianclothes/suits blue.txt /user/include
listdir /westernclothes 1 
updatefile /indianclothes/saree/red.txt /user/include
movefile /indianclothes/suits blue.txt /westernclothes/jeans
unmountvfs database
searchfile blu /tmp
