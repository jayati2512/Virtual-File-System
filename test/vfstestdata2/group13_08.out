createvfs_SUCCESS
mountvfs_SUCCESS
ERR_VFS_MOUNT: VFS_ALREADY_MOUNTED
ERR_VFS_MAKEDIR: VFS_INSUFFICIENT_ARGUMENTS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
ERR_VFS_MAKEDIR: VFS_INSUFFICIENT_ARGUMENTS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
movedir_SUCCESS
ERR_VFS_DELETEDIR: DIRECTORY_IS_NOT_EMPTY
ERR_VFS_ADDFILE:UNABLE_TO_OPEN_SPECIFIED_DATAFILE_PATH
ERR_VFS_ADDFILE:UNABLE_TO_OPEN_SPECIFIED_DATAFILE_PATH
ERR_VFS_LISTDIR: VFS_INSUFFICIENT_ARGUMENTS
ERR_VFS_UPDATEFILE: SOURCE_FILE_PATH_NOT_FOUND
ERR_VFS_MOVEFILE:CANNOT_FIND_SPECIFIED_DESTINATION_PATH
unmountvfs_SUCCESS
ERR_VFS_SEARCHFILE: VFS_NOT_MOUNTED
